app/viewmodels/DutyDefermentAccountViewModel.scala

```diff
@@ -146,20 +148,18 @@ object DutyDefermentAccountViewModel {
           None
         }
 
-        val headWithEntriesOrNoStatement = if (statement.groups.head.monthAndYear.compareTo(monthsToDisplay) > 0) {
+        val headWithEntriesOrNoStatement = if (statement.groups.size > 0) {
```

Comment:
Could you please replace statement.groups.size > 0 with statement.groups.nonEmpty?